src/seasidepropertyhandler.h

```diff
@@ -65,7 +66,7 @@ QTVERSIT_USE_NAMESPACE
 class CONTACTCACHE_EXPORT SeasidePropertyHandler : public QVersitContactHandler
 {
 public:
-    SeasidePropertyHandler();
+    SeasidePropertyHandler(const QStringList &nonexportableProperties = QStringList());
```

Comment:
Is it better to have property names here to remove after generation, or QContactDetail types that we can skip during the export?